### DISCUSSION TITLE
Middleware not working when using 'next build' with output: 'export'

### DISCUSSION THREAD

**Discussion by sarinavaii** [vercel/next.js#54478] [Help]
### Summary

"Hello, I hope you're having a good day.

I'm currently working on publishing my project to a hosting platform and I need to manually export my pages. The project consists of a dashboard that requires a login and access token to load requests. Notably, I have a middleware that functions correctly when deployed on Vercel or on a development server.

In the Next.js documentation, it's mentioned that certain features requiring a Node.js server or involving dynamic logic that can't be computed during the build process are not supported. Middleware falls into this category. I'm keen to understand how to tackle this issue.

Thank you in advance."

`const nextConfig = {
    output: 'export'
};`

**Comment by mehmetalperen** [comment]
@sarinavaii im having the same issue. were you able to solve it? if yes, how?
